lib/WorkflowMain.groovy

```diff
@@ -21,7 +23,7 @@ class WorkflowMain {
     //
     // Generate help string
     //
-    public static String help(workflow, params, log) {
```

Comment:
Why remove the log?

Reply:
That's part of the new template. I'm not sure why they removed that